roles/postgres/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+deployment_type: pulp
+
+#postgres_image: postgres:12
+postgres_image: docker.io/centos/postgresql-96-centos7:9.6
```

Comment:
postgres 9.6 uses `md5`, we are upgrading it to 10 on pulp_installer so we can use `scram-sha-256`. 
Could you please make postgres 10 the default?

Reply:
@fao89 

https://hub.docker.com/r/centos/postgresql-10-centos7 correct?

Reply:
I believe so

Reply:
@mikedep333 can you confirm?

Reply:
I've made this `postgres:12` to align with [awx-operator](https://github.com/ansible/awx-operator/blob/d839088521f0cecd41aebfc772f1c6d936da31e1/roles/installer/defaults/main.yml#L99)

I also set vars to use scram-sha-256:
```
postgres_initdb_args: '--auth-host=scram-sha-256'
postgres_host_auth_method: 'scram-sha-256'
```

Reply:
Is it just about having the ability to use scram-sha-256 (so I shouldn't set it) or should this be the default?

Reply:
I was thinking about the ability, but I like it as default

Reply:
Yeah, I agree with us setting it as the default.